knowledge/responses/why_dont_vegans_eat_honey.md

```diff
@@ -1,5 +1,23 @@
-We don't currently have a response for this myth! Please let us know if you'd like to contribute to writing the response to this myth by posting here: 
+Bee's make honey for bee's, and not for humans.
```

Comment:
I would structure this in point form so it's more readable, for example:

**Bees make honey for bees, not for humans**
[ paragraph about why bees make honey]

**Honey farming is bad for the environment**
[paragraph about honey bees and bumblebees]

**Honey farming practices are cruel**
[if you want, you can add in some stuff about bees being sentient and intelligent and live in a social environment]
[paragraph about how we take honey from them, the substitute we give them is not good enough for their health]
[paragraph about killing bees/hives]
[add a paragraph about how male bees are crushed to get their semen and how queen bees have their wings clipped, you can find this information on earthling ed's why vegans don't eat honey video]

**Alternatives to honey**
[paragraph about plant based syrups and sweeteners]

Other than that I would go through the text and change bee's to bees when it's plural and bee's when it's possessive.

Love the links and the writing, very concise and easy to read!